subprojects/docs/src/snippets/testing/junit-test-suite-categories/groovy/build.gradle

```diff
@@ -0,0 +1,32 @@
+plugins {
+    id 'java'
+}
+
+repositories {
+    mavenCentral()
+}
+
+testing {
+    suites {
+        test {
+            useJUnit()
+
+            dependencies {
```

Comment:
This should be unnecessary, the useJUnit() method should add the deps automatically.